### ISSUE TITLE
Build Canceled: Broccoli Builder ran into an error with `Concat` plugin

### ISSUE THREAD

**Issue by martin-hoger** [Exelord/ember-initials#55]
For of all thank you for the plugin. I very much appreciate your work.

Using version 3.4.0 everything was fine. Upgraded to 3.11.0 and I am getting:
```
Build Canceled: Broccoli Builder ran into an error with `Concat` plugin. 💥
ENOENT: no such file or directory, open '/var/www/tmp/source_map_concat-input_base_path-bAyyG1nq.tmp/node_modules/blueimp-md5/js/md5.js'
Error: ENOENT: no such file or directory, open '/var/www/tmp/source_map_concat-input_base_path-bAyyG1nq.tmp/node_modules/blueimp-md5/js/md5.js'
    at Error (native)
    at Object.fs.openSync (fs.js:641:18)
    at Object.fs.readFileSync (fs.js:509:33)
    at SourceMap.addFile (/var/www/node_modules/fast-sourcemap-concat/lib/source-map.js:74:33)
    at /var/www/node_modules/broccoli-concat/concat.js:200:16
    at Array.forEach (native)
    at Concat.<anonymous> (/var/www/node_modules/broccoli-concat/concat.js:198:24)
    at /var/www/node_modules/fast-sourcemap-concat/lib/source-map.js:416:14
    at SourceMap.end (/var/www/node_modules/fast-sourcemap-concat/lib/source-map.js:403:12)
    at Concat._doLegacyBuild (/var/www/node_modules/broccoli-concat/concat.js:183:22)
```

**Comment by Exelord** [comment]
Thank you for reporting! I will look at it. In the meanwhile did you tried already to remove everything?
I mean: `rm -rf tmp dist node_modules package-lock.json && npm i`

**Comment by Exelord** [comment]
And one more Q; Could you try with `v3.10.0`?

**Comment by martin-hoger** [comment]
Thank you for your quick response. Yes, I tried rm -rf tmp dist node_modules and install, but didn't help. I have also tried v3.10.0 now, the result is the same.

--------------------

npm -v
```
3.10.10
```

ember -v
```
ember-cli: 2.13.2
node: 6.11.0
os: linux x64
```

package.json
```
{
  "name": "XXXX",
  "version": "0.0.0",
  "description": "XXXX",
  "license": "MIT",
  "author": "",
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "repository": "",
  "engines": {
    "node": ">= 4"
  },
  "private": true,
  "scripts": {
    "build": "ember build",
    "start": "ember server",
    "test": "ember test"
  },
  "devDependencies": {
    "@html-next/vertical-collection": "^1.0.0-beta.8",
    "broccoli-asset-rev": "^2.4.5",
    "ember-ajax": "^3.0.0",
    "ember-chrome-devtools": "^0.2.0",
    "ember-cli": "2.13.2",
    "ember-cli-app-version": "^3.0.0",
    "ember-cli-babel": "^6.0.0",
    "ember-cli-dependency-checker": "^1.3.0",
    "ember-cli-eslint": "^3.0.0",
    "ember-cli-htmlbars": "^1.1.1",
    "ember-cli-htmlbars-inline-precompile": "^0.4.0",
    "ember-cli-inject-live-reload": "^1.4.1",
    "ember-cli-mirage": "^0.4.0",
    "ember-cli-moment-shim": "^3.5.0",
    "ember-cli-notifications": "3.2.0",
    "ember-cli-qunit": "^4.0.0",
    "ember-cli-shims": "^1.1.0",
    "ember-cli-sri": "^2.1.0",
    "ember-cli-tutorial-style": "^1.0.1",
    "ember-cli-toggle": "3.0.0",
    "ember-cli-uglify": "^1.2.0",
    "ember-composable-helpers": "^2.0.3",
    "ember-concurrency": "0.7.15",
    "ember-data": "^2.13.0",
    "ember-export-application-global": "^2.0.0",
    "ember-i18n-iso-countries": "^1.4.0",
    "ember-intl": "2.31.1",
    "ember-intl-cp-validations": "2.3.01",
    "ember-cp-validations": "3.5.1",
    "ember-initials": "3.10.0",
    "ember-load-initializers": "^1.0.0",
    "ember-moment": "^7.5.0",
    "ember-pikaday": "2.2.2",
    "ember-power-select": "^1.10.4",
    "ember-resolver": "^4.0.0",
    "ember-route-action-helper": "^2.0.6",
    "ember-sortable": "^1.10.0",
    "ember-source": "~2.13.0",
    "ember-truth-helpers": "^2.0.0",
    "ember-uploader": "1.2.2",
    "ember-world-flags": "^1.0.0",
    "ember-google-charts": "1.5.3",
    "loader.js": "^4.2.3",
    "semantic-ui-ember": "^2.0.1"
  },
}
```

**Comment by Exelord** [comment]
It should be fixed in v3.11.1 :) Please, feel free to reopen
